packages/expo/src/logs/RemoteLogging.ts

```diff
@@ -1,3 +1,4 @@
+import { Platform } from '@unimodules/core';
```

Comment:
We should already import from `expo-modules-core` 😉 but you can leave it as is, I'm going to rename all existing imports soon

Reply:
Ok, let's fix it here as I've already renamed existing imports
```suggestion
import { Platform } from 'expo-modules-core';
```